### ISSUE TITLE
Support AppStream metadata

### ISSUE THREAD

**Issue by n8willis** [raphaelbastide/Unified-Font-Repository#25]
The AppStream specification for package metadata is now used by many (perhaps most) Linux distribution package managers. It now supports font packages as well: https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Fonts.html

Consequently, it would be good if the UFR structure included a skeleton file for this format, too, since it would encourage many open-font developers to get the metadata formatted properly from the get-go.

While the AppStream team has been generating these files for existing packages (see https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Fonts.html ), that's far from ideal. Just including the right sample text is something that the designer is better equipped to do than anyone else....

**Comment by raphaelbastide** [comment]
I think I will redirect visitors of this repo to https://github.com/djrrb/Sample-Font-Repository. It might make some sence to concentrate the efforts on this last project because I have no plan to make UFR evolve in the future.

**Comment by pathumego** [comment]
@n8willis So your suggestion is that we just provide a skeleton `metainfo.xml` file?

**Comment by n8willis** [comment]
@pathumego Yeah. I would contend that having that sort of info in XML has advantages beyond Linux packaging, too, since it's more accessible than having it live solely in the font binary.
